### PR TITLE
[MIG] event_sale: Migration scripts

### PR DESCRIPTION
**Pull request by cubells**
* Analysis work

### CODE REVIEW

File: addons/event_sale/migrations/9.0.1.1/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+event_sale   / event.event              / badge_back (html)             : module is now 'event' ('event_sale')
+# Nothing to do
+
+event_sale   / event.event              / badge_innerleft (html)        : module is now 'event' ('event_sale')
+# Nothing to do
+
+event_sale   / event.event              / badge_innerright (html)       : module is now 'event' ('event_sale')
+# Nothing to do
+
+event_sale   / event.registration       / sale_order_id (many2one)      : NEW relation: sale.order
```
Comment (hbrunn) [inline addons/event_sale/migrations/9.0.1.1/openupgrade_analysis_work.txt:11]: no chance to add those links somehow?

Comment (cubells) [reply addons/event_sale/migrations/9.0.1.1/openupgrade_analysis_work.txt:11]: @hbrunn 
Maybe it works with https://github.com/OCA/OpenUpgrade/pull/713/commits/2de978ad2105bd7ff4d01ec3150297c1efb3d8df

**Review by pedrobaeza — APPROVED** [review]
Squashing on merge

**Comment by pedrobaeza** [comment]
As the request has been satisfied, I merge not so-optimistically, hehe

**Pull request merged**